charts/access/jira/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ include "jira.fullname" . }}
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
Can we abstract these cloud specific annotations and allow custom ones?

I'd look to https://github.com/gravitational/teleport/blob/78e2155268a94100b7763e729a7a70dbe3574e2e/examples/chart/teleport-cluster/templates/service.yaml#L10-L19 as an example here.